### DISCUSSION TITLE
FAILING TO CLONE REPOSITORY ON VS CODE

### DISCUSSION THREAD

**Discussion by MichaelAchugwo** [community/community#62942] [Repositories] [answered]
### Select Topic Area

Question

### Body

Everytime i try to clone a repository on Visual Studio code from my GIthub account it shows an error, "Your account is suspended. please visit support.github.com". When i reached out to support they said my account was not banned.
Does anyone have an idea on how to fix this?

For more clarity, my former Github account i used to use for vs code was actually suspended because the password leaked to hackers who might have misused it and i noticed only when it was suspended and created a new one.

**Comment by elderguardian** [comment, marked as answer]
Hello,

try updating your user to your new account

```
git config --global user.name "YOUR_NEW_NAME"
git config --global user.email "YOUR_NEW_GITHUB_EMAIL"
```

or if you have the github cli

```
gh auth login
```

**Reply by MichaelAchugwo** [reply]
thanks. worked

**Reply by Sokhanetaze80** [reply]
12ib7dApVFvg82TXKycWBNpN8kFyiAN1dr
